### DISCUSSION TITLE
Sponsorship Profile Stuck in "Pending" State – Seeking Advice

### DISCUSSION THREAD

**Discussion by OmarrSakr** [community/community#148077] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hi everyone

I'm currently facing an issue with my GitHub Sponsors profile and was hoping to get .some insights or advice

A few days ago, I received an email from GitHub confirming that my sponsorship :request was approved. The email stated

"!You're now sponsoring @OmarrSakr"
Sponsorship amount: $XX
Next billing date: December 23, 2024

It also encouraged me to share the news on social media, so I assumed everything was set up correctly. However, when I visit my GitHub Sponsors profile, it still shows the status as "Pending approval", and my profile
.isn't live yet
I'm unsure if this is just a delay in the
approval process or a technical issue. Has anyone else experienced something similar? Is there a specific step I might have missed ?after receiving the approval email

Any help or advice would be greatly appreciated. Thank you in advance for your !time and insights
,Best regards
OmarrSakr@

**Comment by divyamlathiya** [comment, marked as answer]
Hey Omarr,

I’ve had a similar issue before, so I can understand how frustrating this might be! It sounds like the email you got means your sponsorship was approved, but sometimes there can be a delay in updating the status on the profile page.

A few things I’d suggest:

1. **Wait a bit:** Sometimes, the system just needs some time to fully update everything after approval. It could be a few hours or up to a day or two before your profile shows as live.

2. **Double-check your setup:** Make sure that all the details in your GitHub Sponsors profile (like your payment info or tax forms) are fully filled out. Sometimes, if something’s missing, it can delay the activation, even after approval.

3. **Contact GitHub Support:** If it’s been more than a couple of days and nothing’s changed, reaching out to GitHub Sponsors support is probably your best bet. They can look into your account specifically and see if there’s an issue on their end.

Hope that helps! Let me know if you need anything else. You’re definitely not alone in dealing with this kind of delay.

Best,
Divyam Lathiya

**Reply by OmarrSakr** [reply]
Hey Divyam,

Thank you so much for your helpful suggestions! I can totally relate to the frustration of delays like this. Your insight into the situation is really appreciated.

I’ll follow your advice and wait a little longer to see if the profile updates. I’ll also double-check that everything in my GitHub Sponsors profile is fully filled out to avoid any issues. If the status doesn’t change in the next couple of days, I’ll reach out to GitHub Sponsors support for further assistance.

Thanks again for taking the time to help, it really means a lot!

Best regards,
Omar

**Reply by divyamlathiya** [reply]
Hey Omar,

You’re very welcome! Glad I could help. Fingers crossed it updates soon! Definitely reach out if it doesn’t – GitHub support should be able to sort it out, follow my account if you have any question.

Good luck, and feel free to reach out if you need anything else!

Best,
Divyam
